app/assets/stylesheets/application.scss

```diff
@@ -1,8 +1,8 @@
 $baseFontSize: 12px;
 
-@import "bootstrap-variables";
 @import "bootstrap";
 @import "bootstrap-responsive";
+@import "bootstrap-variables";
```

Comment:
This change worries me a little. From what I remember, the variables in this file are used to set bootstrap-specific variables, so they would need to be included before the bootstrap pieces. Maybe I'm wrong, or things have changed.